.changeset/stupid-rings-pump.md

```diff
@@ -0,0 +1,22 @@
+---
+'@web3-ui/components': minor
+---
+
+Added the `TokenInput` component which combines an `input` for the user to provide an amount of tokens and a `select` for the user to select the type of token.
+
+```tsx
+const Example = () => {
+  const [value, setValue] = React.useState('');
+  const [token, setToken] = React.useState('');
+
+  return (
+    <TokenInput
+      tokens={['BTC', 'ETH', 'GTC']}
```

Comment:
This does not look like the right API. Aren't you supposed to pass in an array of objects?

Reply:
Yes exactly, I'll have a look to it.